docs: add note about user CA rotation + desktop access

zmb3 opened the pull request:
In the future we'll move desktop access to a separate CA, but for
now Windows certs are generated with the user CA, so Active Directory
will need to be updated to trust the new certificate when a rotation
takes place.

zmb3 commented on docs/pages/setup/operations/ca-rotation.mdx:
```diff
@@ -271,8 +282,17 @@ Host CA  initialized (mode: manual, started: Sep 20 01:44:36 UTC, ending: Sep 21
+  CA pin when adding nodes after rotation.
+</Admonition>
+
+<Admonition type="warning" title="Desktop Access">
```
Is it poor form to have two `Adminition`s back to back like this?

ptgott replied:
I've suggested moving the CA pinning warning since we use it twice in the page.

ptgott commented on docs/pages/setup/operations/ca-rotation.mdx:
```diff
@@ -271,8 +282,17 @@ Host CA  initialized (mode: manual, started: Sep 20 01:44:36 UTC, ending: Sep 21
+
+<Admonition type="warning" title="Desktop Access">
+Teleport signs Windows Desktop certificates with the user certificate authority.
+If the user CA is rotated, the new certificate will need to be
```
Rather than add the link on "exported", which could break the user's focus, I should we should add a "Read more about exporting the Teleport CA" link at the bottom of this admonition box.

zmb3 replied:
I am definitely not the expert, so I defer to you on the final decision, but in this case I think the link makes sense.

As a reader, I'm thinking "you're telling me I have to do this export thing, but I have no idea what that means or how to do it. Why make me search for it?"

zmb3 commented on docs/pages/setup/operations/ca-rotation.mdx:
```diff
@@ -22,35 +22,41 @@ $ tctl get nodes
-Take a look at the [Certificates chapter](../../architecture/authentication.mdx#authentication-in-teleport) in the
-architecture document to learn how the certificate authority rotation works.
+Take a look at the
+[Certificates chapter](../../architecture/authentication.mdx#authentication-in-teleport)
```
I don't know, to be honest. This is a very old page that predates my employment at Teleport. On first glance, I don't see a problem moving this to a "further reading" section.

I guess getting such a thorough review is what I get for wrapping lines that aren't related to the actual change I'm making 😅

zmb3 commented on docs/pages/setup/operations/ca-rotation.mdx:
```diff
@@ -22,35 +22,41 @@ $ tctl get nodes
 
-Administrators can rollback all the changes before rotation is completed by entering `standby`.
+Administrators can rollback all the changes before rotation is completed by
+entering `standby`.
```
How about "by entering the `standby` state"??

We can even include the command required to do so.

zmb3 commented:
@ptgott after thinking on this, many of your comments should be applied to the docs for all recent releases, but the note about desktop access + cert rotation only applies to v8 and up.

I will open a separate PR to address the comments that are not related to desktop access. Once we get that merged and backported to all appropriate branches, I'll update this PR so that it contains only the desktop access change.

Sound reasonable?

ptgott commented:
@zmb3 It does!

zmb3 commented:
@ptgott I've applied most of your suggestions here in PR #10419.

ptgott commented:
@zmb3 Thanks! So my comments here don't overlap too much with my comments in #10419, would you mind removing any changes from this PR that conflict with the other PR?

zmb3 committed:
docs: add note about user CA rotation + desktop access

In the future we'll move desktop access to a separate CA, but for
now Windows certs are generated with the user CA, so Active Directory
will need to be updated to trust the new certificate when a rotation
takes place.

zmb3 commented:
@ptgott I finally got back to this. Should be a much simpler review now that the rest of your comments have already been addressed and merged. PTAL

ptgott commented on docs/pages/desktop-access/getting-started.mdx:
```diff
@@ -148,12 +148,15 @@ certificate-based smart card authentication, and ensuring RDP is enabled.
+Started](../getting-started.mdx) guide.
+
+<Admonition type="note" title="User CA Rotation">
+These steps will need to be repeated if Teleport's user certificate authority is rotated.
```
```suggestion
These steps will need to be repeated if the Teleport's user certificate authority is rotated.
```

zmb3 replied:
hmm. I don't think your suggestion is grammatically correct here.

ptgott replied:
Sorry, I'd meant to change that to "if the Teleport user's," but noticed that the original is fine as it is. Please ignore!

ptgott commented on docs/pages/setup/operations/ca-rotation.mdx:
```diff
@@ -61,11 +61,16 @@ the [customer portal](https://dashboard.gravitational.com/web/login).
+use the *new* CA pin when adding nodes after rotation.
+
+<Admonition type="warning" title="Desktop Access">
+Teleport signes Windows Desktop services with the user certificate authority.
```
```suggestion
Teleport signs Windows Desktop services with the user certificate authority.
```

ptgott commented on docs/pages/setup/operations/ca-rotation.mdx:
```diff
@@ -61,11 +61,16 @@ the [customer portal](https://dashboard.gravitational.com/web/login).
+<Admonition type="warning" title="Desktop Access">
+Teleport signes Windows Desktop services with the user certificate authority.
+If the user CA is rotated, the new CA certificate will need to be exported and
+configured in group policy.
```
Should there be an article/possessive adjective before "group policy"?

zmb3 replied:
No, I don't think so. "Group Policy" is standard terminology in Windows land.

ptgott replied:
👍

ptgott reviewed (approved):
Approved with small comments
